pages.es/common/typeinc.md

```diff
@@ -0,0 +1,21 @@
+# typeinc
+
+> Un programa de prueba de velocidad de tecleo en terminal basado en `ncurses`, escrito en python.
```

Comment:
```suggestion
> Un programa de línea de comandos basado en `ncurses` para probar la velocidad de tecleo, escrito en Python.
```

Reply:
Shouldn't `Python` start with a capital letter on the English page too?

Reply:
Yes

Reply:
Could you change this in the original page @kant?